server/controllers/refer/referralsController.test.ts

```diff
@@ -92,6 +98,29 @@ describe('ReferralsController', () => {
         courseOfferingId,
         pageHeading: "Enter the person's identifier",
       })
+
+      expect(FormUtils.setFieldErrors).toHaveBeenCalledWith(request, response, ['prisonNumber'])
+      expect(response.locals.errors).toEqual(emptyErrorsLocal)
```

Comment:
Let's remove this expectation here, as we're testing that the `response.locals.errors` are set in the `FormUtils` tests.